tests/ff_module-progress.test.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var React = require('react');
+require('./lib/utils').bootstrapBrowser();
+var TestUtils = require('react-addons-test-utils');
+var expect = require('chai').expect;
+
+var Progress = require('../blocks/core/ff_module/ff_module-progress/ff_module-progress.js');
+
+describe('Progress', function() {
+
+  it('should always render a progress bar', function() {
+    var element = React.createElement(Progress, {})
+		var component = TestUtils.renderIntoDocument(element);
+		expect(component).to.exist;
+  })
+
+  it('should always render a progress bar from minimal data', function() {
```

Comment:
I added some more robust tests @dkhuntrods — can you take another look. Ta. (The code isn't very DRY, I didn't know if this was a bad thing in test files like this?)